advisories/github-reviewed/2025/06/GHSA-x5gf-qvw8-r2rm/GHSA-x5gf-qvw8-r2rm.json

```diff
@@ -74,7 +67,7 @@
       "CWE-1333",
       "CWE-400"
     ],
-    "severity": "LOW",
+    "severity": "MODERATE",
```

Comment:
I didn't intend to increase the severity, but the advisory improvement screen forced me to, and I don't see a way to revert this.